spectractor/extractor/targets.py

```diff
@@ -259,25 +268,45 @@ def load(self):
         simbadQuerier = SimbadClass()
         patchSimbadURL(simbadQuerier)
 
-        simbadQuerier.add_votable_fields('flux(U)', 'flux(B)', 'flux(V)', 'flux(R)', 'flux(I)', 'flux(J)', 'sptype',
-                                         'parallax', 'pm', 'z_value')
+        if _USE_NEW_SIMBAD:
+            simbadQuerier.add_votable_fields('U', 'B', 'V', 'R', 'I', 'J', 'sp_type',
+                                             'parallax', 'propermotions', 'rvz_redshift')
+            ra_key = "ra"
+            dec_key = "dec"
+            redshift_key = "rvz_redshift"
+        else:
+            simbadQuerier.add_votable_fields(
+                'flux(U)', 'flux(B)', 'flux(V)', 'flux(R)', 'flux(I)', 'flux(J)', 'sptype',
+                'parallax', 'pm', 'z_value'
+            )
+            ra_key = "RA"
+            dec_key = "DEC"
+            redshift_key = "Z_VALUE"
         if not getCalspec.is_calspec(self.label) and getCalspec.is_calspec(self.label.replace(".", " ")):
             self.label = self.label.replace(".", " ")
         astroquery_label = self.label
         if getCalspec.is_calspec(self.label):
             calspec = getCalspec.Calspec(self.label)
             astroquery_label = calspec.Astroquery_Name
         self.simbad_table = simbadQuerier.query_object(astroquery_label)
+        self.simbad_table.pprint(max_lines=-1, max_width=-1)
```

Comment:
Is this for debugging?

Reply:
Yes. Oops. I was trying to see how the table changed. Will remove the prints and this.